mods/parsoid.js

```diff
@@ -459,7 +460,27 @@ PSP.transformRevision = function (restbase, req, from, to) {
         var body2 = {
             original: original
         };
-        body2[from] = req.body[from];
+        if (from === 'sections') {
+            try {
+                var sections = JSON.parse(req.body.sections);
```

Comment:
Sorry, I probably wasn't clear (and now re-reading my own comment proves it). I just meant that it'd be nice to be flexible so that in case the content type is changed to `application/json` for the code to still be working (I am basing this possibility on the fact that it only takes to change one line in the spec).

> So, I suppose it's enough to add .toString() and check that we've got an object after parsing to protect against smth like sections=1234 where JSON.parse wouldn't throw an exception.

I must admit I don't understand what you mean here. How would `"sections=1234".toString()` change anything?